pyproject.toml

```
@@ -0,0 +1,61 @@
+[project]
+name = "cloud-based-insar-operations"
+requires-python = "==3.10.*"
+dynamic = ["version"]
+
+[tool.setuptools_scm]
+
+# https://docs.pytest.org/en/stable/how-to/mark.html
+[tool.pytest.ini_options]
+addopts = ['--strict-markers']
+markers = [
+    "network: mark tests that require a network connection",
+]
+
+[tool.ruff]
+line-length = 120
+src = [
+    "*",
+    "scripts/*",
+]
+
+[tool.ruff.format]
+indent-style = "space"
+quote-style = "single"
+
+[tool.ruff.lint]
+extend-select = [
+    "I",   # isort: https://docs.astral.sh/ruff/rules/#isort-i
+    "UP",  # pyupgrade: https://docs.astral.sh/ruff/rules/#pyupgrade-up
+    "D",   # pydocstyle: https://docs.astral.sh/ruff/rules/#pydocstyle-d
+    "ANN", # annotations: https://docs.astral.sh/ruff/rules/#flake8-annotations-ann
+    "PTH", # use-pathlib-pth: https://docs.astral.sh/ruff/rules/#flake8-use-pathlib-pth
+]
+
+[tool.ruff.lint.pydocstyle]
+convention = "google"
+
+[tool.ruff.lint.isort]
+case-sensitive = true
+lines-after-imports = 2
+
+[tool.ruff.lint.flake8-annotations]
+suppress-dummy-args = true
+
+[tool.ruff.lint.extend-per-file-ignores]
+"*" = ["D1"]
+
+[tool.mypy]
+python_version = "3.13"
```

Change to match `requires-python = "==3.10.*"` above:
```suggestion
python_version = "3.10"
```